src/Tools/dotnet-monitor/HostBuilderHelper.cs

```diff
@@ -181,8 +181,9 @@ private static void ConfigureGlobalMetrics(IConfigurationBuilder builder)
         private static void ConfigureEndpointInfoSource(IConfigurationBuilder builder, string diagnosticPort)
         {
             DiagnosticPortConnectionMode connectionMode = GetConnectionMode(diagnosticPort);
-            builder.AddInMemoryCollection(new Dictionary<string, string>
-            {
+            builder.AddInMemoryCollection(new Dictionary<string, string> {
+
```

Comment:
Why are newlines added here but not in the other places where list initialization is used, such as a few lines above here? Not complaining that those don't have newlines (I'd rather they don't) but it is inconsistent.